packages/create-plugin/templates/provisioning-template/provisioning/datasources/datasources.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: 1
```

Comment:
The app, datasource and panel templates are based on / copies from the examples repo  "basic" examples where we have provisioning files. Could we make better use of those here to provide actual datasource and dashboards for the scaffolded plugin?

For example when scaffolding a datasource create files based https://github.com/grafana/grafana-plugin-examples/tree/main/examples/datasource-basic/provisioning or when scaffolding a panel create files based on https://github.com/grafana/grafana-plugin-examples/tree/main/examples/panel-basic/provisioning